<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>682</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Разметка зоны</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushBack">
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoadScene">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>38</height>
         </size>
        </property>
        <property name="text">
         <string>Сброс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit1">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>38</height>
         </size>
        </property>
        <property name="text">
         <string>Режим разметки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>38</height>
         </size>
        </property>
        <property name="text">
         <string>Заполнить область</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushStop">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Закончить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditZoneName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Описание</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditZoneDescription"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Размер сегмента</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxSegmentSize"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="DrawingLabel" name="lblMouse">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawingLabel</class>
   <extends>QLabel</extends>
   <header location="global">drawinglabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
